Terraform/variables.tf

```diff
@@ -12,6 +12,11 @@ variable "environment" {
   description = "Environment (Ayuda Dev / Ayuda Preview / Ayuda Cloud)"
 }
 
+variable "resource_group_name" {
```

Comment:
When we add a variable it is required in all tfvar files. We should add it in all files.